### PR TITLE
Feature 6 : Landing Page 

### PR DESCRIPTION
**Pull request by ExtremelySunnyYK**
- [ ] nav bar
- [ ] other points

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,18 +37,24 @@ gem "jbuilder"
...
 
 # Testing
-gem "rspec"
+# gem "rspec"
 
+group :development, :test do
+  gem 'rspec-rails', '~> 5.0.0'
+  # gem 'factory_bot_rails'
+  # gem 'faker'
+  gem 'byebug', '~> 11.1', '>= 11.1.3'
```
Comment (ExtremelySunnyYK) [inline Gemfile:49]: this is a great tool for debugging man! saved me so much time

File: db/seeds.rb [hunk 2]
```diff
@@ -17,9 +17,14 @@
...
     )
     user = User.create(email: "#{i}@gmail.com", password: "123456", name: "User #{i}", lastname: "Lastname #{i}")
-    account = Account.create(user_id: i)
+
+    # Loyalty Program Data
+
 
     #Trans
-    txn = Transaction.create(loyalty_program_data_id: i, amount: i, date: "2000-01-01 02:00:00", status: 0)
+    txn = Transaction.create(loyalty_program_data_id: i, amount: i, date: "2000-01-01 02:00:00", status: 0, receiver: i - 1)
```
Comment (ExtremelySunnyYK) [inline db/seeds.rb:25]: this is still wrong, problem is with creation of LoyaltyProgramData

File: app/models/account.rb [hunk 3]
```diff
@@ -1,9 +1,11 @@
 class Account < ApplicationRecord
   belongs_to :user
-  has_many :loyalty_program_datum, class_name: "loyalty_program_data", foreign_key: "reference_id"
+  has_many :loyalty_program_datum, class_name: 'loyalty_program_data', foreign_key: 'reference_id'
   after_create :create_loyalty_program_data
 
   def create_loyalty_program_data
     # Account.
+    byebug
+    @loyalty_program_data = user.account.build_loyalty_program_datum(points: 0).save
```
Comment (ExtremelySunnyYK) [inline app/models/account.rb:9]: this does not work. need help! @mattChrisP @suvasan19

File: Gemfile [hunk 4]
```diff
@@ -77,9 +83,9 @@ end
...
-  gem "capybara"
-  gem "selenium-webdriver"
-  gem "webdrivers"
+  gem 'capybara'
+  gem 'selenium-webdriver'
+  gem 'webdrivers'
 end
 
-gem "tailwindcss-rails", "~> 2.0"
+gem 'tailwindcss-rails', '~> 2.0'
```
Comment (mattChrisP) [inline Gemfile:91]: I think we can remove the tailwind here right

File: app/models/account.rb (hunk 3 above)
Comment (mattChrisP) [reply app/models/account.rb:9]: Currently I am not too sure about this as well

File: db/seeds.rb (hunk 2 above)
Comment (mattChrisP) [reply db/seeds.rb:25]: Wait what happens when i == 0 what will receiver -1 be referring to?

Comment (ExtremelySunnyYK) [reply db/seeds.rb:25]: thanks for the catch!

**Pull request merged**